files/install-deps-worker.yaml

```diff
@@ -22,11 +22,26 @@
           - python3-bugzilla # python-bugzilla (not bugzilla) on PyPI
           - python3-backoff # Bugzilla class
           #- python3-flask-restx # needs Fedora 32
+          - dnf-utils
+          - python3-pip
+          - make
         state: present
+    - name: Download packit spec file
+      get_url:
+        url: https://raw.githubusercontent.com/packit-service/packit/master/packit.spec
+        # this should be improved - zuul is using var peoject_dir - can be used also in non-zuul
+        # runs - if not set -> set?, will be there any zuul runs conflicts?
+        dest: ./packit.spec
+    - name: Build packit RPM dependencies based on packit.spec
+      command: dnf -y builddep packit.spec
+      become: true
     - name: Install pip deps
       pip:
         name:
           - git+https://github.com/packit-service/sandcastle.git
           - sentry-sdk==0.14.4
           - flask-restx
         executable: pip3
+    - name: Install packit from github with --no-deps
+      # ansible pip doesnt have --no-deps switch
```

Comment:
And why do we actually need the `--no-deps`? If all the deps were installed in previous steps, then they won't be installed in this step anyway. Or is that so much faster with `--no-deps`?

Reply:
we agreed that we will not install any deps via pip and this way we will avoid to install any forgotten depndecy, but if this is ok i also prefer to put it to existing ansible pip install  block

Reply:
Ok, I probably missed that discussion, because I don't remember it.
I don't know what's `'forgotten dependency'`, my view is that:
(as I already said in previous comment) If all deps are installed in the previous `Install packit RPM dependencies` task, then this `pip install` would NOT install any other dependencies.
And if it installs any, then it means that something is missing in the previous step.

Reply:
you are right, will implement it as you suggested

Reply:
But isn't it good if it fails instead of the installation from PyPI when forget to add something to the spec?

We will not get that info when silently installing missing deps from PyPI.

Reply:
Ok, that's a valid point. Then I'm fine with `--no-deps` as long as the reason for using it is documented in the playbook.

Reply:
implemented with --no-deps